### DISCUSSION TITLE
Clarification on uvToRenderTargetUV

### DISCUSSION THREAD

**Discussion by jwwalker** [google/filament#3819] [Q&A] [answered]
The Filament Materials Guide says

> When sampling from a filament::Texture that is attached to a filament::RenderTarget for materials in the surface domain, please use uvToRenderTargetUV to transform the texture coordinate.

The part I don't understand is "surface domain".  I'm thinking this might have something to do with the `vertexDomain` attribute, except that "surface" is not one of the possible values.

**Comment by romainguy** [comment, marked as answer]
The vertex domain is something different. Currently only surface domain materials are exposed but we plan on adding post processing domain materials (we act use those already internally) and maybe other domains (light etc.)
